src/components/investigate-dialog.tsx

```diff
@@ -0,0 +1,62 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./base";
+
+import "./investigate-dialog.sass";
+
+interface IProps extends IBaseProps {}
+interface IState {}
+
+@inject("stores")
+@observer
+export class InvestigateDialogComponent extends BaseComponent<IProps, IState> {
+
+  public render() {
+    return (
+      <div className="investigate-dialog">
+        <div className="investigate-dialog-header">
```

Comment:
Normally I leave out the `investigate-dialog` prefixes here, and scope all my CSS rules so they only apply to the component they're styling. How do you feel about that idea? If we want to have headers like this, that would be an argument for using CSS Modules.

Reply:
In case that wasn't clear, I meant something like this instead:

```
<div className="investigate-dialog">
  <div className="header">
    <div className="title">
...
```

And then in the SASS:

```
.investigate-dialog
  foo: bar
  .header
    baz: qux
    .title
...
```

Reply:
Or perhaps I'm not following.  Could you clarify?

Reply:
Ah sorry no you've got it right! Didn't see your message before I finished writing an example. I don't actually have a strong opinion... @sfentress or @dstrawberrygirl do you?

Reply:
We can try something like that.

Reply:
Yeah, that works fine.  l think it makes it a little harder to search the DOM or figure out where you are when you're inspecting elements (since class names might repeat or be too generic to tell much about an element), but I don't have any major issues with this.  I can easily refactor the SASS with this style.

Reply:
Anyway, I'm fine with either.  Both have advantages/disadvantages.  I can wait for others to chime in.

Reply:
Also, I kind of like the tighter scoping that you presented.  Seems fine (and probably more modern for complex styling).

Reply:
^ I think your points about searching the DOM are good, it's definitely more difficult to select the element you want. Ultimately I think I am happier with the updated styles, since the best way to implement unique selectors would probably still be CSS Modules.

Reply:
I think I'd rather go with the simplified styling idea. A header should have some common properties throughout the app for consistency. If we keep component sass definitions next to the component, as we did with CLUE, then it's clear that the styles we use for a particular component will be used and will override a common base. It's not hard to look up a tree and find the parent that helps pinpoint a particular style definition.  

Some elements will benefit from including a more verbose name - those that are more unique, those that are much less likely to share styling elsewhere. Those cases are fine to be more verbose.

Reply:
Agreed.  I think nesting the classes in Sass is what we settled on in the last update.  Should be less to type and allows generic elements to simply nest within a more uniquely named parent.